blocks/hero/hero.js

```diff
@@ -3,14 +3,25 @@
  * @param {Element} block The hero block element
  */
 export default async function decorate(block) {
+  /* wrap hero block in dedicated section */
+  const section = block.closest('.section');
+  if (section.children.length > 1) {
+    const heroSection = document.createElement('div');
+    heroSection.classList.add('section', 'hero-container');
+    block.closest('main').prepend(heroSection);
+    heroSection.append(block.closest('.hero-wrapper'));
+    section.classList.remove('hero-container');
+  }
+
   if (block.classList.contains('home')) {
     const btnScroll = document.createRange()
       .createContextualFragment(`<div class="btn-scroll">
-<a href="#how-bevespiaerosphere-can-help">Scroll
-<img src="/icons/icon_pink-arrow-down.png" alt="Scroll to section 2 header"/></a>
-</div>`);
+        <a href="#how-bevespiaerosphere-can-help">Scroll
```

Comment:
This is fine for now, we should just keep in mind - and tell authors - that heading IDs depend on the heading text: changing a heading's text would break this anchor's href